modules/combined/tests/contact_verification/patch_tests/ring_3/ring3_mu_0_2_kin.i

```diff
@@ -1,3 +1,9 @@
+[GlobalParams]
+  order = SECOND
+  family = LAGRANGE
+  volumetric_locking_correction = false
```

Comment:
I assume that this is the case you referred to where you turned volumetric locking off to get it to pass with 64 bit indices.  That's not how I'd like to address that issue.  I assume this fails in a way similar to the other tests.  Let's set volumetric_locking_correction = true here, and then skip the test, with a comment that it fails on that platform.